### PR TITLE
fix(ui): team selector and table item alignment

### PR DESCRIPTION
**Pull request by robinrendle**
1. Fixed wacky text alignment/padding of Team Selector on Team Stats page
2. Made spacing between Team Selector + Date Range input more consistent with other pages
3. Fixing the Team selector alignment revealed a bug on the Discover table where items were misaligned. This PR also fixes that.

## Before 

CleanShot 2021-12-07 at 17 27 35@2x

## After
CleanShot 2021-12-07 at 17 27 04@2x

## Before
Group 1849

## After
after29486

### CODE REVIEW

File: static/app/components/idBadge/baseBadge.tsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ const DisplayNameAndDescription = styled('div')`
 const DisplayName = styled('span')`
   overflow: hidden;
   text-overflow: ellipsis;
-  line-height: 1.2;
```
Comment (robinrendle) [inline static/app/components/idBadge/baseBadge.tsx:82]: This shouldn’t have an effect on the other instances, will double check the snapshots. But this is why the text was misaligned when the font-size property is changed.

File: static/app/views/organizationStats/teamInsights/overview.tsx [hunk 2]
```diff
@@ -352,7 +352,7 @@ const Body = styled(Layout.Body)`
 const ControlsWrapper = styled('div')`
   display: grid;
   align-items: center;
-  gap: ${space(1)};
+  gap: ${space(2)};
```
Comment (evanpurkhiser) [inline static/app/views/organizationStats/teamInsights/overview.tsx:355]: I just changedhttps://github.com/getsentry/sentry/pull/30490 wait I just changed releases spacing here.

`space(1)` seems more consistent with what the rest of the app uses no?

Comment (robinrendle) [reply static/app/views/organizationStats/teamInsights/overview.tsx:355]: Nah, it should be `space(2)`— Team Stats, Issues, and Alerts are the outliers. We’ll get around to fixing it there, too. Releases should stick to `space(2)` horizontal/vertical gap as well.

**Review by scttcper — APPROVED** [review]
big gap

File: static/app/components/group/releaseStats.tsx [hunk 3]
```diff
@@ -157,8 +157,3 @@ export default memo(GroupReleaseStats);
 const TooltipWrapper = styled('span')`
   margin-left: ${space(0.5)};
 `;
-
-const StyledIconQuest = styled(IconQuestion)`
```
Comment (robinrendle) [inline static/app/components/group/releaseStats.tsx:161]: Not needed now because it's properly aligned with flex.

**Pull request merged**